telosevm.tokenlist.json

```diff
@@ -198,6 +198,17 @@
          "telosevm"         
         ]
       }
+       {
```

Comment:
This is invalid JSON, there needs to be a comma following the previous object in the array before this new object you've added.